src/View/CrudView.php

```diff
@@ -6,6 +6,32 @@
 
 class CrudView extends View
 {
+    public $layout = 'CrudView.default';
+
+    public function initialize(array $config = [])
```

Comment:
You're extending the core's `View::initialize()` which doesn't require an argument - that's what I meant. But since it was copy/pasted from BootstrapUI's README where the mistake is initially.

In any case, it's resolved in my changes.